resources/event-sources/values.yaml

```diff
@@ -4,9 +4,9 @@ global:
   controllerManager:
     name: controller-manager
     image: event-sources-controller-manager
-    dir: develop/
-    version: #TODO
+    dir: ''
+    version: dc48c27c
   httpAdapter:
     image: event-sources-http-adapter
-    dir: develop/
-    version: #TODO
+    dir: ''
+    version: 0b31784e
```

Comment:
there is also this version: [dc48c27c](https://console.cloud.google.com/gcr/images/kyma-project/EU/event-sources-http-adapter@sha256:44b3283569284d9b31584ad30da839a118ebcee7b07de7531aac4e20f9c40471/details?tab=info)` for the event-sources-http-adapter
I think - even if no changes to this particular code have been introduced - we should use this as it is the latest version

Reply:
Done